toolchain/tools/llvm_release_name.py

```diff
@@ -87,7 +87,10 @@ def _linux(llvm_version):
             os_name = "linux-gnu-ubuntu-20.04"
             
     elif (distname == "ubuntu" and version.startswith("18.04")) or (distname == "linuxmint" and version.startswith("19")):
-        os_name = "linux-gnu-ubuntu-18.04"
+        if major_llvm_version >= 12:
+            os_name = "linux-gnu-ubuntu-16.04"
+        else:
+            os_name = "linux-gnu-ubuntu-18.04"
     elif (distname == "ubuntu" and version.startswith("20")) or (distname == "pop" and version.startswith("20")):
         # use ubuntu 18.04 clang LLVM release for ubuntu 20.04
         os_name = "linux-gnu-ubuntu-18.04"
```

Comment:
We probably want to use the real 20.4 release here for LLVM >= 12?

Reply:
That's a good point, thanks for catching that :)

Reply:
On second look, this seems to be redundant with the clause starting at line 80, a few lines above this?

Reply:
Yep, you're right. I've cleaned this up and minimized the diff.